packs/spell-effects/spell-effect-bullhorn.json

```diff
@@ -4,7 +4,7 @@
     "name": "Spell Effect: Bullhorn",
     "system": {
         "description": {
-            "value": ""
+            "value": "<p>Granted by <em>@UUID[Compendium.pf2e.spells-srd.Item.Bullhorn]</em></p>\n<p>You gain a +1 status bonus to checks to @UUID[Compendium.pf2e.actionspf2e.Item.Coerce] and auditory Performance checks to Perform at a large venue.</p>"
```

Comment:
```suggestion
            "value": "<p>Granted by @UUID[Compendium.pf2e.spells-srd.Item.Bullhorn]</p>\n<p>You gain a +1 status bonus to checks to @UUID[Compendium.pf2e.actionspf2e.Item.Coerce] and auditory Performance checks to Perform at a large venue.</p>"
```

Reply:
Oops, merged already. Well, it's fine we have a few other emphasized spell links I've caught so there may be some cleanup to do with it.